en/log/flight_review.md

```diff
@@ -0,0 +1,332 @@
+
+> **Tip** For the rate controller in particular, it is useful to enable the high-rate logging profile ([SDLOG_PROFILE](../advanced/parameter_reference.md#SDLOG_PROFILE)) to get more details when zooming in.
+
+
+## Vibration
+
+Vibration is one of the most common problems for multirotor vehicles.
+High vibration levels can lead to:
+- less efficient flight and reduced flight time
+- the motors can heat up
+- increased material wearout
+- inability to tune the vehicle tightly, resulting in degraded flight performance.
+- sensor clipping
+- position estimation failures, potentially resulting in fly-aways.
+
+It is therefore important to keep an eye on the vibration levels and improve the setup if needed.
+
+There is a point where vibration levels are clearly too high, and generally lower vibration levels are better. 
+However there is a broad range between 'everything is ok' and 'the levels are too high'.
+This range depends on a number of factors, including vehicle size - as larger vehicles have higher inertia, allowing for more software filtering (at the same time the vibrations on larger vehicles are of lower frequency).
+
+The following paragraphs and sections provide information about what plots to use for checking vibration levels, and how to analyse them.
+
+> **Tip** It is worth looking at multiple charts when analysing vibration (different charts can better highlight some issues).
+
+
+### Actuator Controls FFT
+
+> **Note** You need to enable the high-rate logging profile ([SDLOG_PROFILE](../advanced/parameter_reference.md#SDLOG_PROFILE)) to see this plot.
+
+This graph shows a frequency plot for the roll, pitch and yaw axis based on the actuator controls signal (the PID output from the rate controller). 
+It helps to identify frequency peaks and configuring the software filters.
+There should only be a single peak at the lowest end up to a few Hz, the rest should be low and flat.
+
+Note that the y-axis scaling is different for different vehicles, but logs from the same vehicle can be directly compared to each other.
+
+#### Examples: Good Vibration
+
+[QAV-R 5" Racer](https://docs.px4.io/en/frames_multicopter/qav_r_5_kiss_esc_racer.html) frame (excellent vibration).
+Note that this allows us to considerably increase the cutoff frequency of the software filters
+
+![Low vibration QAV-R 5 Racer - FFT plot](../../assets/flight_log_analysis/flight_review/vibrations_good_actuator_controls_fft.png)
+
+DJI F450 frame (good vibration).
+Note the blade passing frequency of the propellers around 100 Hz.
+
+![Low vibration DJI F450 - FFT plot](../../assets/flight_log_analysis/flight_review/vibrations_f450_actuator_controls_fft.png)
+
+S500 frame:
+
+![Low vibration S500 actuator controls - FFT plot](../../assets/flight_log_analysis/flight_review/vibrations_s500_actuator_controls_fft.png)
```

Comment:
Do we need to think about the roll/pitch/yaw mean values in images?

Reply:
If you want we can explain it: they should be a help when comparing logs with different filter settings.